sdk/python/lib/pulumi/resource.py

```diff
@@ -342,57 +344,68 @@ def __init__(self,
                     raise Exception(
                         "'depends_on' was passed a value that was not a Resource.")
 
+    def _merge_instance(self, opts: 'ResourceOptions') -> 'ResourceOptions':
```

Comment:
How do doc comments work in this situation?  Do we need to include doc comments on this `_merge_instance` that will get picked up and displayed for the instance `merge`?

Reply:
It will not.  But i've updated this PR to rectify that.  Good catch!